server/server.js

```diff
@@ -35,9 +35,7 @@ const keycloak = require('./keycloak.js');
 const apiBaseUrl = '/api/v1';
 const app = express();
 
-if (process.env.NODE_ENV === 'local') {
-  app.use(cors());
-}
+app.use(cors());
```

Comment:
This should be reverted once FE and BE are on same OCP cluster.

Reply:
Is this true now?

Reply:
This is not yet true. GDX has one DNS record left to swap over. There's a ticket that captures this but I don't see it as super high priority.